src/Components/FlashSalesCountdownSection/FlashSalesCountdownSection.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import Countdown from "./Countdown/Countdown";
+import ArrowButtons from "./ArrowButtons/ArrowButtons";
+import styles from "./FlashSalesCountdownSection.module.css";
+import Title from "./Title/Title";
+import Heading from "./Heading/Heading";
+const SecondSection = () => {
+  return (
+    <div className={styles.flexContainer}>
+      <div className={styles.container}>
+        <Title Title="Today's" />
+        <Heading Heading="Flash Sales" />
+      </div>
+      <div className={styles.container}>
+        <div className={`${styles.flexContainer} ${styles.Countdown}`}>
+          <Countdown days={3} hours={23} minutes={19} seconds={56} />
```

Comment:
Everything in the file is good, but what if I am going to use the component in other places? I will be forces to use render the countdown, right?
What about conditionally rendering it?

Reply:
Add props to TitleSection to control the visibility of Countdown component.

For example in App:

import React, { useState } from "react";
import TitleSection from "./TitleSection/TitleSection";

const App = () => {
  const [showCountdown] = useState(true);

  return (
    <>
      <TitleSection showCountdown={showCountdown} />
    </>
  );
};

export default App;